pinc/filter_project_list.inc

```diff
@@ -241,10 +245,7 @@ function process_and_display_project_filter_form($username, $filter_type, $filte
 
     $difficulty_element->echo_html_control();
 
-    echo "<input type='submit' name='reset' value='", attr_safe(_("Remove Filter")), "'>";
-    echo "&nbsp;<input type='submit' name='apply' value='", attr_safe(_("Apply Selections")), "'>";
-    echo "</fieldset>";
-    echo "</form>\n";
+    echo "<input type='submit' name='apply' value='", attr_safe(_("Apply Selections")), "'>";
```

Comment:
Not important, but I learned today that you can use `,` to do multiple echos as mentioned here https://stackoverflow.com/questions/1466408/difference-between-period-and-comma-when-concatenating-with-echo-versus-return

Reply:
I only recently discovered that myself after @70ray started using them. Most of the code does concatenation and we've got a bit of a mixed bag going on.